app/src/androidTest/java/com/duckduckgo/app/analyticsSurrogates/AnalyticsSurrogatesLoaderTest.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2018 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.analyticsSurrogates
```

Comment:
Not keen on the name. Is this analytics as in google? The surrogates is more generic than that, it's just that's the main one we're targeting now.

Also a personal thing, but really dislike camel naming in packages but that's because I'm used to the Java conventions, though it seems Kotlin allows it (but discourages two words in a package name if possible).

Just surrogates would do imho :) 

For reference: 

* https://kotlinlang.org/docs/reference/coding-conventions.html
* https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

Reply:
Renaming to simply _surrogates_